terraform/environments/planetfm/locals_security_groups.tf

```diff
@@ -22,6 +22,40 @@ locals {
           security_groups = []
         }
       }
+    }
+
+    Migration_cutover_sg = {
```

Comment:
no capital M, _sg not needed, unsure about name

Reply:
it's important to copy the form of what already exists

Reply:
yes its a temporary security group

Reply:
Ive removed the Capital and sg. The ctuover is in reference to it being used temporarily for a migration cutiover instance